docs/docs/getting-started/v3-migration.md

```diff
@@ -156,6 +156,8 @@ options: {
 }
 ```
 
+Also, the time scale option `distribution: 'series'` was removed and a new chart type `timeseries` was introduced in its place.
```

Comment:
[nit] I think this should be "new scale type"

Reply:
fixed